### ISSUE TITLE
What's the BIOS version of your setup?

### ISSUE THREAD

**Issue by BillDH2k** [NOTNlCE/Dell-Precision-T5810-OpenCore#1]
Thanks for sharing your working EFI. Could you clarify the BIOS version of your T5810 setup? Also, do you really need CpuTscSync for this platform? 

THANKS!

**Comment by NOTNlCE** [comment]
I believe I am on the latest BIOS available from Dell. I can dig the tower out and give you the exact number later today.

**Comment by BillDH2k** [comment]
Thanks. Very few reported their success with Big Sur for this Dell system. Noticed that you used older version OC 0.6.4. Any particular reason why? Also, your patches are relatively simple, did you just follow the standard OpenCore setup?

Thanks!

**Comment by NOTNlCE** [comment]
I developed this OC EFI many months ago when OC 0.6.4 was the latest. I likely have an updated copy I can commit. OC is slated to release 0.8.0 later today, so I will update on the box and commit the newest release and kexts. This system is my "emergency spare" so it doesn't see regular use. 

Yes - a very basic set of patches and SSDTs in this repo. I had tested with several additional ACPI edits but they led to some system instability such as reboots during sleep. All appears functional with the minimal patch set here, including sleep, though that may differ depending on your CPU/GPU setup. 

Notably macOS Monterey does not boot with this EFI. I had tried a few months back when I updated the EFI. I suspect there are some changes needed to be made, but I have not had the time to investigate further. System is currently on the latest release of macOS Big Sur.

**Comment by BillDH2k** [comment]
Thank you for detailed reply. One observation for your EFI, it boots perfectly fine for an existing Big Sur install (copied from another system, Z420). But the picker failed to show "Big Sur Installation" from the USB stick. Not sure what adjustment need to be made for the config.plist. So right now, I can't do a fresh install, but only boot from an existing install.

Any idea?

Thanks!

**Comment by NOTNlCE** [comment]
This repository is missing the HfsPlus.efi driver, therefore not seeing your HFS+ formatted installation media. I will correct it in the new commit. You can resolve manually by adding HfsPlus.efi from OCBinaryData or OpenHfsPlus.efi from newer OpenCore to your Drivers folder and add the appropriate entry in your config.plist.

**Comment by BillDH2k** [comment]
I added OpenHfsPlus.efi instead and it worked! This EFI also worked on Catalina (11.15.7) too! The only issue I encountered was when I tried to enable the verbose mode (adding "-v" to boot-args), I got KP on "PCI scanning ..." error. In non-verbose mode, everything works fine. I was abled to fix this error by adding "PCIRootUID=1" to the boot-args (a googled answer).

This is a much cleaner EFI compared to the one I used earlier (which consists of lots of spaghetti patches, appeared to  be converted from a dsdt patch, with Catalina support only!)

Thanks a lot for your great work!
